packages/repository/src/model.ts

```diff
@@ -211,8 +211,9 @@ export abstract class Model {
     };
 
     const json: AnyObject = {};
+    const hiddenProperties: Array<string> = def.settings.hiddenProperties || []; // ToDo (frbuceta): Short-term solution
```

Comment:
The comment does not make much sense to me:
- if it's a todo, then let's make it actionable 
- if you just want to note that this is a short-term solution, then use `NOTE(frbuceta)` instead of `TODO`

Personally, I would leave the comment out, I don't see much value in it.

Reply:
I will remove the comment